pkg/daemon/ceph/client/config.go

```diff
@@ -133,6 +140,48 @@ func generateConfigFile(context *clusterd.Context, clusterInfo *ClusterInfo, pat
 	return filePath, nil
 }
 
+func mergeDefaultConfigWithRookConfigOverride(context *clusterd.Context, clusterInfo *ClusterInfo, configFile *ini.File) error {
+	cm, err := context.Clientset.CoreV1().ConfigMaps(clusterInfo.Namespace).Get(k8sutil.ConfigOverrideName, metav1.GetOptions{})
+	if err != nil {
+		if !kerrors.IsNotFound(err) {
+			return errors.Wrapf(err, "failed to read configmap %s", k8sutil.ConfigOverrideName)
+		}
+		return nil
+	}
+
+	config, ok := cm.Data["config"]
+	if !ok {
+		return nil
+	}
+
+	cfg, err := ini.Load([]byte(config))
+	if err != nil {
+		return errors.Wrapf(err, "failed to load config data from %s", k8sutil.ConfigOverrideName)
+	}
+
+	for _, section := range cfg.Sections() {
```

Comment:
Is there not an existing ini helper method that would merge for us? For example, we want to merge the results of `createGlobalConfigFileSection()` with the ceph.conf that is mounted to a path as previously mentioned.

Reply:
In fact it seems that there is an Append method that I missed. Will look into it